internal/stage_helpers.go

```diff
@@ -278,7 +279,11 @@ func serveTrackerResponse(w http.ResponseWriter, r *http.Request, responseConten
 	queryParams := r.URL.Query()
 	left := queryParams.Get("left")
 	if left == "" {
-		logger.Errorln("Required parameter missing: left")
+		if isMagnetLinkTest {
+			logger.Errorln("Required parameter \"left\" is missing in peers request. Use a placeholder value like left=999 for magnet links as file size is not known in advance.")
```

Comment:
Should we suggest `999` or `1`? Not sure if it's considered an error if `left` is greater than the size of the torrent file.